### ISSUE TITLE
Change all players inventories to that a specific group

### ISSUE THREAD

**Issue by weeryan17** [Multiverse/Multiverse-Inventories#348]
My server is going to be updating to the snapshots as they have some features we really need, and I need to set the inventories of all players saved on the world file to that of the default (our survival mode) group. I was working on my own converter to do this, but I realized it might be faster if there was a way via the plugin to do this. Is there a way via the plugin to set all player inventories to that of a specific group?

**Comment by nicegamer7** [comment]
Sorry, I'm not sure I understand your request. Can you rephrase it?

**Comment by weeryan17** [comment]
Basically I changed over to vanilla and was wondering if there was a way to set all the inventories to that of an inventory group.

As an example I have a survival group, and a creative group. As the main world is survival I wanted, when I go to vanilla, the actual inventories of the players to be the survival inventory without requiring them to log in a go to the survival world.

**Comment by nicegamer7** [comment]
Okay, I think I understand, like an export command?

**Comment by weeryan17** [comment]
Basically yegh.

**Comment by nicegamer7** [comment]
Okay, I'm going to close this as a duplicate of #88. Please check it out and vote!
